src/test/java/org/apache/drillui/test/framework/pages/QueryResultsPage.java

```diff
@@ -37,6 +39,106 @@
   @FindBy(xpath = "//*[@title=\"Open in new window\"]")
   private WebElement profileButton;
 
+  @FindBy(xpath = "/html/body/div[2]/div[2]/table/tbody/tr/td[1]/button/span[1]")
+  private WebElement queryResultStatus;
+
+  @FindBy(id = "delimitBy")
+  private WebElement exportCSVDelimiter;
+
+  @FindBy(xpath = "/html/body/div[2]/div[2]/table/tbody/tr/td[3]/button")
+  private WebElement exportButton;
+
+  @FindBy(xpath = "//*[@id=\"result_filter\"]/label/input")
+  private WebElement rowsFilter;
+
+  @FindBy(xpath = "//*[@id=\"result_wrapper\"]/div[1]/div[2]/button")
```

Comment:
Find buttons by text: 
```suggestion
  @FindBy(xpath = "//button[contains(text(), 'Export')]")
```